apis/stardust/api-spec/spec.json

```diff
@@ -1590,6 +1596,12 @@
           "optional": true,
           "kind": "function",
           "params": []
+        },
+        "focusSelection": {
```

Comment:
Added in [PR](https://github.com/qlik-oss/nebula.js/pull/701/files#diff-cf1484a8b521c2fdc3949ef8fc6cd1859e12740ef65da0202e4e9c47704be5f8R1032)